components/style/servo/selector_parser.rs

```diff
@@ -100,44 +100,55 @@ impl PseudoElement {
 #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 #[allow(missing_docs)]
 pub enum NonTSPseudoClass {
-    AnyLink,
-    Link,
-    Visited,
     Active,
+    AnyLink,
+    Checked,
+    Disabled,
+    Enabled,
     Focus,
     Fullscreen,
     Hover,
-    Enabled,
-    Disabled,
-    Checked,
     Indeterminate,
-    ServoNonZeroBorder,
+    Lang(String),
```

Comment:
Sooner or later we're going to have to implement https://tools.ietf.org/html/rfc4647#section-3.3.2, storing a String here would just ease the implementation of it. Not sure if a `Box<str>` would achieve the same effect.